### PR TITLE
SearchView controller without callback implementation.

### PR DESCRIPTION
**Pull request by ZPendragon**
Basic setup for SearchViewController. Pushing update shortly because Git lost my SearchViewController.h

### CODE REVIEW

**Comment by ZPendragon** [comment]
There are three expected errors in SearchViewController.m since I explicitly split the PR from the manager & model implementation.

File: Spotify-ObjC/Spotify-ObjC/SAArtist.h [hunk 1]
```diff
@@ -0,0 +1,13 @@
+//
+//  SAArtist.h
+//  Spotify-ObjC
+//
+//  Created by Kevin Zeckser on 8/1/16.
+//  Copyright © 2016 Kevin Zeckser. All rights reserved.
+//
+
+#ifndef SAArtist_h
```
Comment (petey3) [inline Spotify-ObjC/Spotify-ObjC/SAArtist.h]: You don't need these, same comment for other classes

File: Spotify-ObjC/Spotify-ObjC/SASearchViewController.m [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// MARK: - Setup
+
+- (void)setupVC {
+    UIColor *backgroundColor = [UIColor blackColor];
+    UIColor *placeholderTextColor = [UIColor whiteColor];
+    self.view.backgroundColor = backgroundColor;
+    self.textField.backgroundColor = backgroundColor;
+    self.navigationController.navigationBar.barTintColor = backgroundColor;
+    self.navigationController.navigationBar.backgroundColor = backgroundColor;
+    _textField.attributedPlaceholder = [[NSAttributedString alloc] initWithString:@"Search Spotify" attributes:@{NSForegroundColorAttributeName: placeholderTextColor}];
```
Comment (petey3) [inline Spotify-ObjC/Spotify-ObjC/SASearchViewController.m]: style: use `self.textField`

File: Spotify-ObjC/Spotify-ObjC/SASearchViewController.m [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    } else {
+        // Run our query
+    }
+    return YES;
+}
+
+// MARK: - UITableViewDataSource
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return 10;
```
Comment (petey3) [inline Spotify-ObjC/Spotify-ObjC/SASearchViewController.m:56]: Is this 10 just because there isn't a data source yet?

Comment (ZPendragon) [inline Spotify-ObjC/Spotify-ObjC/SASearchViewController.m:56]: Yes.

**Comment by petey3** [comment]
Made a couple of comments

A more general comment would be it probably would have made more sense to make this an entirely new project and repo instead of deleting all the old stuff on this branch but thats also just the weird circumstance of this mini-project so, no worries. Just a note for next time something like this happens

**Comment by petey3** [comment]
Anyways good to "🚢 " once you take out the weird ifndef statements

**Pull request merged**